tools/fds/server.cpp

```diff
@@ -111,11 +130,10 @@ int main (
         rtps.setName(is.str().c_str());
     }
 
-    fastdds::dds::Log::SetCategoryFilter(
-        std::regex("(RTPS_PDPSERVER_TRIM)|(RTPS_PARTICIPANT)|(DISCOVERY_SERVER)"
-        "|(SERVER_PDP_THREAD)|(CLIENT_PDP_THREAD)|(RTPS_WRITER_HISTORY)|(RTPS_READER_HISTORY)|(RTPS_PDP)"
-        "|(DISCOVERY_DATABASE)|(RTPS_PDP_LISTENER)|(SERVER_PING_THREAD)"));
-    fastdds::dds::Log::SetVerbosity(fastdds::dds::Log::Kind::Info);
+    //  fastdds::dds::Log::SetCategoryFilter(
+    //      std::regex("(RTPS_HISTORY)|(RTPS_WRITER_HISTORY)|(RTPS_READER_HISTORY)|(RTPS_PDP_SERVER)|(READER_PROXY)"
+    //      "|(RTPS_PDP)|(SERVER_PDP_THREAD)|(CLIENT_PDP_THREAD)|(DISCOVERY_DATABASE)|(RTPS_PDP_LISTENER)"));
+    //  fastdds::dds::Log::SetVerbosity(fastdds::dds::Log::Kind::Info);
```

Comment:
Remove this block

Reply:
Somehow I feel I'm gonna miss it the next bug hunt.🤠